pkgs/games/classicube/default.nix

```
@@ -64,6 +64,7 @@ stdenv.mkDerivation rec {
   postBuild = "cd -";
 
   installPhase = ''
+    runHook preInstall
     mkdir -p "$out/bin"
     cp 'src/ClassiCube' "$out/bin"
```

there's already a desktop file in the archive

```suggestion
    mkdir -p "$out/bin" "$out/share/applications"
    cp 'src/ClassiCube' "$out/bin"
    cp 'classicube.desktop' "$out/share/applications/"
```